src/overlays/ovl_i4/fox_ka.c

```diff
@@ -1,63 +1,2183 @@
+                    actor_it->obj.pos.z = dest.z;
+                    actor_it->unk_0F4.y = actor->unk_04E * 18.0f;
+                    actor_it->state = 1;
+                    actor_it->unk_0E4 = i + 10;
+                    actor_it->unk_0E6 = D_i4_8019F444[i];
+                    actor_it->unk_0F4.x = 3.0f;
+                    actor_it->health = 24;
+                    if (actor_it->unk_0B6 >= 2) {
+                        actor_it->health = 1000;
+                    }
+                    actor_it->iwork[11] = 1;
+                    actor_it->unk_0C9 = 1;
+                    if (D_i4_801A0540 < 9600) {
+                        actor_it->unk_044 = 22;
+                    }
+                    actor_it->timer_0C2 = 30;
+                    Object_SetInfo(&actor_it->info, actor_it->obj.id);
+                    if (actor_it->unk_0B6 == 1) {
+                        actor_it->info.unk_1C = 0.0f;
+                        actor_it->info.bonus = 0;
+                        Audio_PlaySfx(0x3100000C, actor_it->sfxPos, 4, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+                    }
+                    actor_it->info.action = func_i4_80198AA0;
+                    actor_it->info.draw = func_i4_801995B4;
+                    actor->unk_04E++;
+                }
+                break;
+            }
+        }
+    }
+}
+
+void func_i4_8019848C(void) {
+    Actor* actor = &gActors[9];
+
+    Actor_Initialize(actor);
+    actor->obj.status = 2;
+    actor->obj.pos.x = 0;
+    actor->obj.pos.y = 1000.0f;
+    actor->state = 1;
+    actor->unk_0E4 = 9;
+    actor->unk_0E6 = 27;
+    actor->unk_0B6 = 2;
+    actor->health = 1000;
+    actor->iwork[11] = 1;
+    actor->unk_0C9 = 1;
+    actor->timer_0C2 = 30;
+    actor->obj.id = OBJ_ACTOR_197;
+    Object_SetInfo(&actor->info, actor->obj.id);
+    actor->info.unk_1C = 0.0f;
+    actor->info.bonus = 0;
+    Audio_PlaySfx(0x3100000CU, actor->sfxPos, 4, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+}
+
+void func_i4_80198594(Actor* actor) {
+    s32 pad[4];
+    f32 D_i4_8019F494[5] = { -200.0f, -100.0f, -0.0f, 100.0f, 200.0f };
+
+    switch (actor->state) {
+        case 0:
+            D_80161A44 = 30000.0f;
+            D_i4_801A0540 = 0;
+            D_8015F920 = D_8015F921 = 0;
+            actor->state = 2;
+            if (D_8015F924 != 0) {
+                gHitCount = gSavedHitCount;
+                gBosses[1].state = 6;
+                gBosses[1].obj.pos.x = 0.0f;
+                gBosses[1].obj.pos.z = 0.0f;
+                gBosses[1].obj.pos.y = 2000.0f;
+                Audio_PlaySfx(0x11037025U, gBosses[1].sfxPos, 0, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+                D_8015F928 = 20000;
+                gBosses[1].swork[16] = 5760;
+                D_8015F920 = 1;
+                func_8001D444(0, 32797, 0, 255);
+            }
+
+            func_i4_8019848C();
+
+            PRINTF("KT_time %d\n", D_i4_801A0540);
+
+        case 2:
+            D_i4_801A0540 += 1;
```

Comment:
```suggestion
            D_i4_801A0540++;
```
Also that PRINTF seems more likely to be here than in the previous case.